webservices/docs.py

```diff
@@ -510,6 +510,10 @@
 AFFILIATED_COMMITTEE_NAME = '''
 Affiliated committee or connected organization
 '''
+
+IS_COMMITTEE_ACTIVE = '''
+Affiliated committee or connected organization
+'''
 # ======== committee end ===========
 
```

Comment:
Should the description of IS_COMMITTEE_ACTIVE be different than the one of AFFILIATED_COMMITTEE_NAME?  @fec-jli